aip/adopting.md

```diff
@@ -46,6 +38,10 @@ repository. Once you've done that, you can cite all AIPs specifically using
 that domain name (e.g., `aip.example.com/1234`) and you'll always get sent to
 the right place.
 
+While it's possible to create your own AIP IDs, it is recommended that you
+maintain the same AIP IDs that are used in aip.dev. This will allow your AIPs
+to be comparable across multiple AIP implementations.
+
```

Comment:
There's a lot in this PR that could benefit from the spec idea I discussed in another PR, but for me this is the primary reason. If we have to ask adopters nicely to follow specific rules in order to keep things compatible, then I don't think it will actually happen.